hack/clean-ci.sh

```diff
@@ -18,41 +18,18 @@ set -o errexit  # exits immediately on any unexpected error (does not bypass tra
 set -o nounset  # will error if variables are used without first being defined
 set -o pipefail # any non-zero exit code in a piped command causes the pipeline to fail with that code
 
-export PATH=${PWD}/hack/tools/bin:${PATH}
-REPO_ROOT=$(git rev-parse --show-toplevel)
-
-# shellcheck source=./hack/ensure-kubectl.sh
-source "${REPO_ROOT}/hack/ensure-kubectl.sh"
-
-on_exit() {
-  # kill the VPN
-  docker kill vpn
-}
-
-trap on_exit EXIT
-
-# Run the vpn client in container
-docker run --rm -d --name vpn -v "${HOME}/.openvpn/:${HOME}/.openvpn/" \
-  -w "${HOME}/.openvpn/" --cap-add=NET_ADMIN --net=host --device=/dev/net/tun \
-  gcr.io/k8s-staging-capi-vsphere/extra/openvpn:latest
-
-# Tail the vpn logs
-docker logs vpn
-
-  # Wait until the VPN connection is active.
-function wait_for_vpn_up() {
-  local n=0
-  until [ $n -ge 30 ]; do
-    curl "https://${GOVC_URL}" --connect-timeout 2 -k && RET=$? || RET=$?
-    if [[ "$RET" -eq 0 ]]; then
-      break
-    fi
-    n=$((n + 1))
-    sleep 1
-  done
-  return "$RET"
-}
-wait_for_vpn_up
+# Fallback for mirror-prow.
```

Comment:
Not saying it necessarily has to continue to work, but does this still work without VPN?

Reply:
yes, only upstream prow with VMC required VPN.